src/DynamicRupture/FrictionLaws/GpuImpl/LinearSlipWeakening.h

```diff
@@ -245,7 +245,7 @@ class LinearSlipWeakeningLaw
         auto ltsFace = item.get_group().get_group_id(0);
         auto pointIndex = item.get_local_id(0);
 
-        real resampledSlipRate = SpecializationT::resampleSlipRate(
+        real const resampledSlipRate = SpecializationT::resampleSlipRate(
```

Comment:
Can we add `QualifierAlignment: Left` to `.clang-format` please?

Reply:
Absolutely.

Reply:
Implemented and propagated now.